src/session/user.rs

```diff
@@ -319,4 +380,33 @@ impl User {
     pub fn status_expression(user_expression: &gtk::Expression) -> gtk::Expression {
         gtk::PropertyExpression::new(User::static_type(), Some(user_expression), "status").upcast()
     }
+
+    pub fn formatted_status_expression(user_expression: &gtk::Expression) -> gtk::Expression {
+        let type_expression = User::type_expression(user_expression);
+        let status_expression = User::status_expression(user_expression);
+        gtk::ClosureExpression::new(
+            |args| {
+                let user_status = &args[1].get::<BoxedUserStatus>().unwrap().0;
+                let type_ = &args[2].get::<BoxedUserType>().unwrap().0;
+                match type_ {
+                    UserType::Regular => match user_status {
```

Comment:
Actually, you could make this case `UserType::Regular | UserType::Deleted` instead of "hardcoding" the "last seen a long time ago" string here.

Another solution would be to show "deleted user" as a subtitle for deleted accounts (which makes more sense) and is the same behavior as Telegram X. Although we intend to follow what the current official clients are doing to keep it consistent. This is just an idea and should be discussed first.

Reply:
Is Telegram X still alive 😄? 

>  instead of "hardcoding" the "last seen a long time ago" string for UserType::Deleted here.

I don't think it's "hardcoding" since official client does the same thing.  Showing "deleted user" might be the solution might but Telegram X is almost dead and most Telegram Desktop users are probably familar with "last seen a long time ago", so it's better to leave it like that.

Reply:
> Another solution would be to show "deleted user" as a subtitle for deleted accounts

Since we already have "Deleted Account" as the title it's a bit weird to show the same information also in the subtitle. I think we can either follow the official clients showing the "last seen a long time ago" string or just don't show a subtitle. I don't have a particular preference honestly.